boards/mips/pic32mz/chipkit-wifire/README.txt

```diff
@@ -0,0 +1,60 @@
+README
+======
+
+  chipKIT Wi-FIRE board
+  (https://chipkit.net/wiki/index.php?title=ChipKIT_Wi-Fire) features
+  the Microchip PIC32MZ2048EFG100 MCU, Microchip MRF24WG0MA Wi-Fi module,
+  USB 2.0 Full-Speed / Hi-Speed OTG controller, Micro SD card connector etc.
+  The PIC32MZ2048EFG100 is a 200 MHz 32-bit MIPS M5150 operation with 2MB Flash
+  memory and 512KB RAM.
+
+Contents
+========
+
+  - GCC toolchain installation
+  - Building NuttX for the chipKIT Wi-FIRE board
+  - Programming Flash
+  - Serial console
+
+GCC toolchain installation
+==========================
+
+  $ wget https://sourcery.mentor.com/GNUToolchain/package12725/public/mips-sde-elf/mips-2014.05-24-mips-sde-elf-i686-pc-linux-gnu.tar.bz2
+  $ sudo tar xvjf mips-2014.05-24-mips-sde-elf-i686-pc-linux-gnu.tar.bz2 -C /usr/local
+
+Building NuttX for the chipKIT Wi-FIRE board
+============================================
+
+  $ cd nuttx
+  $ make apps_distclean
+  $ make distclean
+  $ ./tools/configure.sh -l chipkit-wifire:nsh
+  $ make CROSSDEV=mips-sde-elf-
```

Comment:
Why don't we add this toolchain to arch/mips/src/mips32/Toolchain.defs?

Reply:
I've added it to Toolchain.defs and Kconfig, and updated the instructions in README.txt. Thanks.